javascript/chronometer.js

```diff
@@ -1,26 +1,40 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
-  startClick(callback) {
-    // ... your code goes here
+  startClick(printTime) {
```

Comment:
the callback function printTime needed to be called inside the interval, as it is mentioned in the instructions.